config/config.go

```diff
@@ -144,14 +144,15 @@ type Config struct {
 
 	RefusePayload multiFlag `yaml:"refuse-payload"`
 
+	// KubernetesIngressV1 setting is deprecated and has no effect, only v1 is supported
+	KubernetesIngressV1 bool `yaml:"kubernetes-ingress-v1"`
```

Comment:
I think we can just remove it

Reply:
done in all the places (skipper, config and dataclient Options)